Add docs for getting intel_gpu_top to work without privileged mode (#6166)
blakeblackshear/frigate: docs/docs/configuration/hardware_acceleration.md

  hwaccel_args: preset-rpi-64-h264
```

### Intel-based CPUs

> #### Intel-based CPUs (<10th Generation) via VAAPI

Reviewer: ```suggestion
#### Via VAAPI
```

Reviewer: unrelated to your PR, but this would be a good change

**NOTICE**: With some of the processors, like the J4125, the default driver `iHD` doesn't seem to work correctly for hardware acceleration. You may need to change the driver to `i965` by adding the following environment variable `LIBVA_DRIVER_NAME=i965` to your docker-compose file or [in the frigate.yml for HA OS users](advanced.md#environment_vars).

> #### Intel-based CPUs (>=10th Generation) via Quicksync

Reviewer: ```suggestion
####  Via Quicksync (>=10th Generation only)
```

Reviewer: same as above

QSV must be set specifically based on the video encoding of the stream.

##### H.264 streams

```yaml
ffmpeg:
  hwaccel_args: preset-intel-qsv-h264
```

##### H.265 streams

```yaml
ffmpeg:
  hwaccel_args: preset-intel-qsv-h265
```

#### Docker Configuration

> Additional configuration is needed for the Docker container to be able to access the `intel_gpu_top` command for GPU stats. Two possible changes need to be made: 1) Adding the `CAP_PERFMON` capability and 2) Setting the `perf_event_paranoid` low enough to allow access to the perfomance event system.

Reviewer: ```suggestion
Additional configuration is needed for the Docker container to be able to access the `intel_gpu_top` command for GPU stats. Three possible changes can be made:
1. Run the container as privileged.
2. Adding the `CAP_PERFMON` capability.
3. Setting the `perf_event_paranoid` low enough to allow access to the perfomance event system.
```

> 

Reviewer: add an example for privileged

##### CAP_PERFMON

Only recent versions of Docker support the `CAP_PERFMON` capability. You can test to see if yours supports it by running: `docker run --cap-add=CAP_PERFMON hello-world`

Pass `cap-add=CAP_PERFMON` to the container. With docker compose, that looks like this:

```yaml
services:
  frigate:
    ...
      cap_add:
        - CAP_PERFMON

> ```

Reviewer: I would refactor this to show specific examples of both similar to how the nvidia docs show docker compose and docker cli examples
